### DISCUSSION TITLE
Crose Tab

### DISCUSSION THREAD

**Discussion by QusaiImayyah** [QuestPDF/QuestPDF#377] [Q&A] [answered]
How to write crose tab?

**Comment by girlpunk** [comment]
Are you able to provide an example of what a crose tab is?

**Reply by QusaiImayyah** [reply]
![image](https://user-images.githubusercontent.com/20271178/196143472-f81dfbcd-0906-4945-b010-575bc90a6a88.png)
This is crosstab , which is about dynamic columns.
 I discovered that it could be built

**Reply by girlpunk** [reply]
This should be possible with the [table](https://www.questpdf.com/api-reference/table) element. You will likely have to calculate the details of the columns by iterating through row objects ahead of time for the relevant `ColumnsDefinition` calls.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Regarding the table functionality you asked about, you can find detailed documentation and examples here:
https://www.questpdf.com/api-reference/table/basics.html

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
